src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1915,10 +1900,13 @@ namespace WaterToAirHeatPumpSimple {
 		SimpleWatertoAirHP( HPNum ).AirMassFlowRate = PLRCorrLoadSideMdot;
 
```

Comment:
OK, so here's the problem. This model expects to use design water flow rate. In Init, the call to SetComponentFlowRate can return a value other than design. Sometimes it is design, and other times not. So using the node flow rate is troublesome. I chose to use the design water flow rate here and multiply by PLR and then use that value and the actual Qsource to calculate outlet water temp and enthalpy a few lines down.